Makefile

```diff
@@ -13,6 +13,9 @@ PATH := $(PATH):$(GOPATH)/bin
 # OS
 UNAME := $(shell uname)
 
+MIXPANEL_PROJECT_TOKEN := 3fae49de272be1ceb8cf34119f747073
+ACCOUNT_TOKEN := lilico:sF60s3wughJBmNh2
```

Comment:
is this for mainnet ?

Reply:
yes

Reply:
don't even get me started... apparently, there's no other way if use homebrew... since homebrew builts its own binaries and hence can not access the secrets we use in CD.